doc/languages-frameworks/python.md

```diff
@@ -165,7 +165,7 @@ run the script in the `python3` shell.
 
 ```py
 #! /usr/bin/env nix-shell
-#! nix-shell -i 'python3.withPackages(ps: [ps.numpy])'
+#! nix-shell -i python3 -p 'python3.withPackages(ps: [ps.numpy])'
```

Comment:
@Mic92 I think we shouldn't advertise anymore `-p python3 -p python3.numpy` because that works because of the shell hook which, in my opinion, is something for `nix-shell` development, and not for just creating an environment.

Reply:
well. but this approach does not work either:

```console
$ cat ./test
#! /usr/bin/env nix-shell
#! nix-shell -i python3 -p 'python3.withPackages(ps: [ps.numpy])'
import numpy
$ ./test
error: syntax error, unexpected $undefined, at (string):1:94
```

Reply:
Oops, indeed. The quotes are incorrect.